department-of-veterans-affairs/abd-vro
# MCP-2156: Fix python consumers

## What was the problem?

Some python consumers were not configured to log timestamps. 

Associated tickets or Slack threads:

- [MCP-2156](https://amida.atlassian.net/browse/MCP-2156)

## How does this fix it?[^1]

Logging setup is now applied to python consumers
## How to test this PR
- Step 1 build and run project and check the python consumer's logs
- Step 2[^secrel]


TO DO: add testing

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
